packages/server-core/src/user/identity-provider/identity-provider.hooks.ts

```diff
@@ -311,20 +333,7 @@ export default {
     ],
     find: [
       iff(isProvider('external'), setLoggedinUserInQuery('userId')),
-      // Combined check for external provider and $like usage with scope verification
-      iff(
-        isProvider('external') &&
-          ((context) =>
-            context.params.query?.accountIdentifier?.$like !== undefined ||
-            context.params.query?.email?.$like !== undefined), // Use iffElse to check for admin action
-        iffElse(
-          isAction('admin'),
-          // If the action is admin, check for admin or super scope
-          verifyScope('admin', 'super'),
-          // If not admin action, check for editor:write scope
-          verifyScope('editor', 'write')
-        )
-      )
+      iff(isProvider('external') && validateQueryParameters, verifyScope('admin', 'admin'))
```

Comment:
I dont think we can use `&& validateQueryParameters` like this. I think it need a hook context 

https://github.com/ir-engine/ir-engine/blob/cd401f049ea35a2ee7477db8f17d6ef4c65a331e/packages/server-core/src/networking/instance/instance.hooks.ts#L157

Reply:
@hanzlamateen updated.

Reply:
Please also write unit test to verify this hook.

I doubt if it works because if you see my referenced `packages/server-core/src/hooks/is-action.ts` hook it returns a delegate.

Tests will help verify.